changelog/pending/20250508--programgen-python--add-scalar-return-method-call-programgen-support-to-python.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: programgen/python
+  description: Add scalar return method call programgen support to python 
```

Comment:
It's a nit but I think it's valuable for us to be consistent here (it surely matters less in the changelog, but for us internally and the code/docs we write I think it's important) -- it's not technically scalar, because you could e.g. return a list, or a union -- it's just not necessarily object.